modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -369,9 +370,8 @@ void QueueReader::rewindToStop(std::vector<Q*>   &in_queues,
     ::rewindToStop(in_queues, this_id);
 }
 
-bool QueueReader::getInputVector(std::vector<Q*> &in_queues,
-                                 cv::GRunArgs    &in_constants,
-                                 cv::GRunArgs    &isl_inputs)
+cv::gimpl::StreamMsg QueueReader::getInputVector(std::vector<Q*> &in_queues,
```

Comment:
can you clarify - why  it changed  on returning value?

Reply:
Previously `bool` means `!= Stop{}`, now except `Stop` there can be `Exception` message, so it was changed from bool to `StreamMsg`